src/SDK/SDK.php

```diff
@@ -128,14 +128,14 @@ public function __construct(Language $language, Spec $spec)
         $this->twig->addFilter(new TwigFilter('comment2', function ($value) {
             $value = explode("\n", $value);
             foreach ($value as $key => $line) {
-                $value[$key] = "         * " . wordwrap($value[$key], 75, "\n         * ");
+                $value[$key] = "     * " . wordwrap($value[$key], 75, "\n     * ");
```

Comment:
Are we sure this doesn't affect other langs?

Reply:
Yes, `comment2` and `comment3` is only used in the Web SDK 👍🏻